docs/node-operators/migration-guides/upgrade-to-0710.md

```diff
@@ -0,0 +1,401 @@
+
+
+This guide is specifically intended for those who are already running a validator node with version `v0.` or higher.
+
+Before you start, note that the instructions use the following variables for file paths:
+
+* `<VEGA-NETWORK-HOME>`: the home path to the Vega network, e.g., `/home/vega/vega_home`
+* `<TENDERMINT-HOME>`: the Tendermint home path, e.g., `/home/vega/tendermint_home`
+* `<VEGAVISOR-HOME>`: the Vega Visor home path, e.g., `/home/vega/vegavisor_home`
+* `<BACKUP-FOLDER>`: the folder where you store backups, e.g., `/home/vega/backups`
+* `<VISOR-BIN>`: the path to the Vega Visor binary, e.g., `/home/vega/bin/visor`
+* `<VEGA-BIN>`: the path to the Vega core binary, e.g., `/home/vega/bin/vega`
+* `<CHAIN-ID>`: new chain ID for network; it is required to pass as an argument for data node, e.g., current chain ID on mainnet is: `vega-mainnet-0009`
+
+The following are placeholders for the PostgreSQL connection details for the data node - the ones you put in the data node `config.toml`.
+
+* `<VEGA-DB-USER>` - PostgreSQL user you create and put in the data node config
+* `<VEGA-DB-NAME>` - PostgreSQL database name
+
+This guide will refer to the above paths. The sample paths given above are just examples. We recommend setting the paths that align with the conventions adopted by your organisation.
+
+## Study the changes between versions
+Before upgrading your node software, review the **[changelog](https://github.com/vegaprotocol/vega/blob/develop/CHANGELOG.md#0710)** for a list of breaking API changes for each version from the previously released version, and find links to each release's detailed changelog.
+
+## Upgrade steps
+
+### 1. Stop the network
+At this point, validators need to choose and agree on the checkpoint that will be loaded into the network during the next restart, and stop the network as soon as everyone agrees on the selected checkpoint.
+
+The reason to quickly stop the network is to avoid producing more checkpoints and missing transactions after the restart (the transactions executed between when the selected checkpoint is produced, and the network is stopped).
+
+For testnet releases, the time requirement may be relaxed, but mainnet downtime must be limited as much as possible.
+
+1. Stop the running node
+    - Stop `Vega` and optionally `data node` if you are not running Visor: `sudo systemctl stop vega && sudo systemctl stop data-node`
+    - Stop `vegavisor` if you are running your node with Visor: `sudo systemctl stop vegavisor`
+2. Verify the node has been stopped:
+    - Vega: `systemctl status vega`
+    - Optionally data node: `systemctl status data-node`
+    - Optionally Visor: `systemctl status vegavisor`
+
+### 2. Create backup
+
+```bash
+mkdir -p <BACKUP-FOLDER>/v0.70.3/wallets;
+mkdir -p <BACKUP-FOLDER>/v0.70.3/core-state;
+mkdir -p <BACKUP-FOLDER>/v0.70.3/tm-state;
+
```

Comment:
In previous versions the backup folder had the **upgrade to** version in the file path - so I'm guessing these were just missed, but I think they should be 0.71.1?